tests/integration/test_global.py

```diff
@@ -545,6 +545,67 @@ def test_install_twice(pixi: Path, tmp_path: Path, dummy_channel_1: str) -> None
     assert dummy_b.is_file()
 
 
+def test_install_twice_with_force_reinstall(
+    pixi: Path, tmp_path: Path, dummy_channel_1: str, dummy_channel_2: str
+) -> None:
+    env = {"PIXI_HOME": str(tmp_path)}
+
+    dummy_b = tmp_path / "bin" / exec_extension("dummy-b")
+
+    # Install dummy-b
+    verify_cli_command(
+        [
+            pixi,
+            "global",
+            "install",
+            "--channel",
+            dummy_channel_1,
+            "dummy-b",
+        ],
+        env=env,
+    )
+    assert dummy_b.is_file()
+
+    # Modify dummy-b channel and try to install it again
+    # it will not change anything
```

Comment:
```suggestion
    # Modify dummy-b channel and try to install it again
    # Even though we changed the channels, it will claim the environment is up-to-date
```